x-pack/platform/plugins/private/upgrade_assistant/public/application/components/es_deprecations/deprecation_types/cluster_settings/table_row.tsx

```diff
@@ -88,7 +90,11 @@ export const ClusterSettingsTableRow: React.FunctionComponent<Props> = ({
   ]);
 
   return (
-    <>
+    <EuiTableRow
+      data-test-subj="deprecationTableRow"
+      key={`deprecation-row-${index}`}
+      onClick={() => setShowFlyout(true)}
```

Comment:
Now that the `onClick` event handler is added to each `EuiTableRow`, do we still need to pass the `openFlyout={() => setShowFlyout(true)}` prop to `EsDeprecationsTableCells`? I think its used for the issue name link but probably not needed now given that the whole row is clickeable